### ISSUE TITLE
Suggestion: Add lighter blue color for more dynamic syntax

### ISSUE THREAD

**Issue by jinh0** [pineapplegiant/spaceduck#41]
Example: VSCode default color scheme
<img width="605" alt="Screen Shot 2021-05-31 at 12 01 07 PM" src="https://user-images.githubusercontent.com/40512164/120133900-f2437980-c207-11eb-8545-5361b7cb01ef.png">

**Comment by pineapplegiant** [comment]
I like this screenshot. I think bluegreen and light blue, or maybe a darker blue would be nice for sure.

**Comment by bigpick** [comment]
# `#00bbf9`
![00bbf9_add](https://user-images.githubusercontent.com/9803299/121049695-f2062980-c785-11eb-8b2c-160bf39008dd.png)

# `#14CFFB`
![14CFFB_add](https://user-images.githubusercontent.com/9803299/121049699-f3375680-c785-11eb-9ede-82757062cab3.png)

---

Though, I think a lighter shade of blue/purple would probably suite the theme more? Granted, it's not _blue_ persay, but still... e,g

# `#BCC4DB`
![BCC4DB_add](https://user-images.githubusercontent.com/9803299/121050070-45787780-c786-11eb-9961-a4a240c5580f.png)

# `#85C7F2`
![85C7F2_add](https://user-images.githubusercontent.com/9803299/121050347-7eb0e780-c786-11eb-8126-bfd313afb8ee.png)



Of the blue-blues though, I think `#14CFFB` looks better in terms of matching the theme; though`#85C7F2` looks nice for a blue-blue alternative too.

**Comment by pineapplegiant** [comment]
I was thinking a darker color maybe?

Although, lot's of things going on at once..(like I wanna edit both the red and orange too)

We haven't written it anywhere explicitly yet, but we updated the current cyan/blue color. My apologies, but `#00A3CC` is old and we're currently using `#59C2FF` for [the new cyan on dev](https://github.com/pineapplegiant/spaceduck/blob/4c509053876e72e69a45a24367df557c398e1639/colors/spaceduck.vim#L34) and I think I like it to make it official on main. (We'd have to update a lot of things I know 😢 ). It might make sense to call that the light blue honestly now that I look at it. @jinh0 added this color lol

I was thinking just adding a turqouise and a darker blue maybe, what do you think? 🤔 
**Turquoise:** `#49D1BD`
**Maybe new darkBlue:** `#278FCC`

[Link to colors on coolors.co](https://coolors.co/e33400-e39400-5ccc96-f2ce00-b3a1e6-7a5ccc-30365f-59c2ff-278fcc-49d1bd)

![spaceduck](https://user-images.githubusercontent.com/32819563/121053597-bfa1ff80-c778-11eb-86e9-55b3406a627e.png)

Side note, I love how github generates the little color bubbles next to the hex -> `#30365F`. Absolutely beautiful.

**Comment by bigpick** [comment]
Ah, OK. That's cool, makes sense.

As for the coloring, I like the new cyan/turquoise/"dark blue", but I think the key is making sure their expected usage is clearly defined in the spec. Like, the "main vibe" to me of the theme is a dark purple grounded theme, with purple accents and pops of color; that way in things that aren't like exact syntax that you can define a hard spec for, it keeps the overall "vibe" consistent (like the atom-ui colors, etc).

Kinda curious what you're thinking for the red/orange now too though hehe

**Comment by pineapplegiant** [comment]
Yeah I think that’s it. I tend to associate red with errors so it should be used least sparingly I think. But orange is more permissible just not as much as green/purple.
